src/Agent.Plugins/TfsVCCliManager.cs

```diff
@@ -172,6 +182,25 @@ protected async Task<string> RunPorcelainCommandAsync(FormatFlags formatFlags, p
             return string.Join(Environment.NewLine, result.Output ?? new List<string>());
         }
 
+        protected async Task<TfsVCPorcelainCommandResult> TryRunPorcelainCommandAsync(FormatFlags formatFlags, int retriesOnFailure, params string[] args)
+        {
+            var result = await TryRunPorcelainCommandAsync(formatFlags, args);
+            int attempt = 0;
+            while (attempt < retriesOnFailure && result.Exception != null)
```

Comment:
`attempt` isn't used outside the loop, so this can be a usual for loop.
```
for (int attempt = 0; attempt < retriesOnFailure && result.Exception != null; attempt++)
```